<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences</class>
 <widget class="QDialog" name="Preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Einstellungen</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>120</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>120</height>
      </size>
     </property>
     <property name="title">
      <string>Konfiguration der Prüflinge und Korrektoren</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="horizontalSpacing">
         <number>22</number>
        </property>
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Anonyme Prüflinge</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QFormLayout" name="formLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Anzahl</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spinBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="value">
             <number>16</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="2">
         <layout class="QFormLayout" name="formLayout_7">
          <item row="0" column="0">
           <widget class="QLabel" name="label_10">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Muster</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Prüfling #</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkBox_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Anonyme Korrektoren</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QFormLayout" name="formLayout_6">
          <item row="0" column="0">
           <widget class="QLabel" name="label_9">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Anzahl</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spinBox_2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="value">
             <number>6</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="2">
         <layout class="QFormLayout" name="formLayout_8">
          <item row="0" column="0">
           <widget class="QLabel" name="label_11">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Muster</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Korrektor #</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Einstellung der Prüfungszeiten (gilt für beide Tage gleichermaßen)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Vormittags</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <layout class="QFormLayout" name="formLayout_3">
          <property name="formAlignment">
           <set>Qt::AlignCenter</set>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Prüfungsdauer</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spinBoxDuration2">
            <property name="suffix">
             <string> Minuten</string>
            </property>
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>180</number>
            </property>
            <property name="singleStep">
             <number>5</number>
            </property>
            <property name="value">
             <number>60</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Nachmittags</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QFormLayout" name="formLayout_4">
          <property name="formAlignment">
           <set>Qt::AlignCenter</set>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>2. Beginn</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QTimeEdit" name="timeEditBegin2">
            <property name="time">
             <time>
              <hour>14</hour>
              <minute>0</minute>
              <second>0</second>
             </time>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QFormLayout" name="formLayout">
          <property name="formAlignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="verticalSpacing">
           <number>-1</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>1. Beginn</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QTimeEdit" name="timeEditBegin1">
            <property name="showGroupSeparator" stdset="0">
             <bool>false</bool>
            </property>
            <property name="dateTime">
             <datetime>
              <hour>9</hour>
              <minute>0</minute>
              <second>0</second>
              <year>1999</year>
              <month>12</month>
              <day>31</day>
             </datetime>
            </property>
            <property name="time">
             <time>
              <hour>9</hour>
              <minute>0</minute>
              <second>0</second>
             </time>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="2">
         <layout class="QFormLayout" name="formLayout_2">
          <property name="formAlignment">
           <set>Qt::AlignCenter</set>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Prüfungsdauer</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spinBoxDuration1">
            <property name="suffix">
             <string> Minuten</string>
            </property>
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>180</number>
            </property>
            <property name="singleStep">
             <number>5</number>
            </property>
            <property name="value">
             <number>60</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelLunch">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Mittagspause:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidgetTimes">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>90</height>
         </size>
        </property>
        <row>
         <property name="text">
          <string>Tag 1</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Tag 2</string>
         </property>
        </row>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Preferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Preferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
